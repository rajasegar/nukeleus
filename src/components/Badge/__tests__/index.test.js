import React from 'react';
import { renderWithTheme } from '../../../test-utils';

import { Badge } from '../index';

describe('Badge component test suite', () => {
  it('Should render Badge', async () => {
    const { container } = renderWithTheme(<Badge> 90 </Badge>);
    expect(container.firstChild).toMatchSnapshot();
  });
});
