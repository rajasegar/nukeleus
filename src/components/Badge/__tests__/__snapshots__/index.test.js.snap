// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Badge component test suite Should render Badge 1`] = `
.c0 {
  height: 16px;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  font-weight: 600;
  font-size: 12px;
  line-height: 14px;
  text-align: center;
  border-radius: 10px;
  padding: 1px 8px;
  color: #FFFFFF;
  background: #2C5CC5;
}

<span
  class="c0"
  type="primary"
>
   90 
</span>
`;
